app/views/patients/_patient_dashboard.html.erb

```diff
@@ -1,33 +1,33 @@
 <div id="patient_dashboard_content">
   <%= bootstrap_form_for patient, html: { id: 'patient_dashboard_form' }, remote: true do |f| %>
     <div class="col-sm-4" >
-      <%= f.text_field :name, label: 'First and last name', autocomplete: 'off' %>
-      <%= f.text_field :primary_phone, value: patient.primary_phone_display, label: 'Phone number', autocomplete: 'off' %>
+      <%= f.text_field :name, label: t('patient.dashboard.name'), autocomplete: 'off' %>
```

Comment:
Some of these keys we're going to reuse in a few different places. I wonder about having a form namespace explicitly for form labels and stuff like that which might be the same across different forms on different views? @lomky @DarthHater do you have any thoughts?

Reply:
I'll create on called 'shared' and move name and status under there. we can shift other things into it as we realize they're needed in multiple places. :tada: